src/config.py

```diff
@@ -98,6 +121,21 @@ class TestConfig(Config):
     BRIDGE_IP_ADDRESS = '127.0.0.1'
     USERNAME = 'test-user'
 
+    POLICY_HUE_MAPPING = {
```

Comment:
TestConfig would inherit from the default in Config, but I overrode it to make sure the unit test cases consistently work. More specifically, in the case a user modified the POLICY_HUE_MAPPING in Config the tests would possibly stop working since the tests are based off the values that are in the dictionary right now, but by overriding the dict in TestConfig, the tests would continue to work even in the case that the dict in Config was changed.

Reply:
I see, I think it's fine either way, whichever you decide. If you override, maybe leave a comment explaining the decision?

Reply:
Sounds good, just added the comment